app/assets/javascripts/discourse/components/topic-list-item.js.es6

```diff
@@ -35,6 +35,34 @@ export const ListItemDefaults = {
   attributeBindings: ["data-topic-id"],
   "data-topic-id": Ember.computed.alias("topic.id"),
 
+  init() {
+    this._super(...arguments);
+
+    if (typeof this.get("topic.read_by_group_member") !== "undefined") {
+      this.messageBus.subscribe(
+        `/private-messages/group-read/${this.get("topic.id")}`,
+        data => {
+          const node = document.querySelectorAll(
```

Comment:
Might be wrong but I think 

```
document.querySelector(`.indicator-topic-${data.topic_id}`)
```

would do the same

Reply:
Thanks, @jjaffeux <3